ICAFIX/hcp_fix_multi_run

```diff
@@ -825,6 +825,13 @@ then
     rm -f ${concatfmri}_Atlas.dtseries.nii ${concatfmri}_Atlas_hp${hp}.dtseries.nii
 fi
 
+# Convert sICA+FIX cleaned movement regressors to text
+if [ -f ${concatfmrihp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean.nii.gz ] ; then
+  fslmaths ${concatfmrihp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean.nii.gz -Tmean -mul 0 -add 1 ${concatfmrihp}.ica/mc/prefiltered_func_data_mcf_conf_hp_clean_mask.nii.gz
```

Comment:
This looks like the "mask" is an all-ones volume (not a brain mask or nonzero variance mask), is this intended, and if so, why is it needed at all?

Reply:
Agreed.  Not sure why L830 is there.

Reply:
It is not clear that fslmeants will work without a mask or coordinate, that is why there is a mask.

Reply:
Doesn't seem hard to test, and if it worked, it would entirely remove several lines (some of which were initially confusing).  Doesn't crash on an anatomical image, though it does take many seconds on fsl 5:
```
tim@timsdev:/mnt/myelin/tim/voltest$ fslmeants -i testLPI.nii --showall | tail -c 500
63 2.391223 2.394073 2.396367 2.397355 2.397202 2.396351 2.395081 2.393568 2.392322 2.392496 2.395739 2.403646 2.417208 2.436579 2.461160 2.489789 2.520775 2.551845 2.580199 2.602878 2.617405 2.622426 2.618012 2.605447 2.586682 2.563829 2.539038 2.514791 2.494272 2.481347 2.479870 2.492495 2.519450 2.557961 2.602637 2.646768 2.684010 2.709814 2.722121 2.721240 2.709162 2.688713 2.662853 2.634269 2.605276 2.577890 2.553897 2.534770 2.521444 2.513808 2.510787 2.510695 2.511874 2.513183 2.514090
```
It appears to be outputting voxel indices first, followed by the values, as its output word count is exactly 4 times the number of voxels.